src/ralph/admin/views/multiadd.py

```diff
@@ -34,7 +39,15 @@ def get_form(self):
             'model': self.model
         }
         for field in self.fields:
-            multi_form_attrs[field] = MultilineField(allow_duplicates=False)
+            field_type = self.model._meta.get_field(field)
+            if isinstance(field_type, models.IntegerField):
+                multi_form_attrs[field] = IntegerMultilineField(
+                    allow_duplicates=False
```

Comment:
it should be possible to add multiple items with the same position (ex. blade servers), so allow_duplicates should be set to True here sometimes. Is it possible to specify it on field level?